.github/workflows/build_packages.yaml

```diff
@@ -6,9 +6,11 @@ concurrency:
 
 on:
   workflow_run:
-    workflows: ["Run fvt tests","Run test cases"]
+    workflows: [ "Run fvt tests","Run test cases" ]
```

Comment:
Will this cause the workflow to be triggered repeatedly?

Reply:
So they are `And` relationships and it only gets triggered when both workflows("Run fvt tests" and "Run test cases") complete?

Reply:
Yes, this is what we expected.

Reply:
Cool, looks good for me